src/apps/loan-list/materials/stackable-material/stackable-material.tsx

```diff
@@ -48,7 +48,7 @@ const StackableMaterial: FC<StackableMaterialProps & MaterialProps> = ({
       )}
       role="button"
       onClick={handleOpenDueDateModal}
-      onKeyUp={(e) => {
+      onKeyDown={(e) => {
```

Comment:
please make sure to test this then holding the button down for a bit to see how it reacts

Reply:
Good point, there’s something I need to look into before merging.